torch/csrc/jit/passes/frozen_ops_to_mkldnn.cpp

```diff
@@ -205,6 +213,41 @@ void InplaceMKLDNNSubgraph(std::shared_ptr<Graph> graph) {
   }
 }
 
+Operation createUnaryOp(
+    std::function<void(at::Tensor output, at::Tensor input)> aten_op,
+    std::function<Tensor(Tensor)> fallback) {
+  return [aten_op, fallback](Stack* stack) {
+    auto a = pop(stack).toTensor();
+    if (a.numel() == 0) {
+      // clone should be cheap for a 0-size tensor
+      push(stack, a.clone());
+    }
+    if (a.is_mkldnn()) {
+      c10::impl::ExcludeDispatchKeyGuard edkg(c10::autograd_dispatch_keyset);
+      auto a_it = at::native::itensor_from_mkldnn(a);
+      auto raw_data = a_it.get_data_handle();
+      auto topt = a.options().layout(c10::kStrided);
+      auto t = at::from_blob(raw_data, {a.numel()}, topt);
+
+      auto it_empty = ideep::tensor(a_it.get_desc());
```

Comment:
im not really familiar with this code but have we verified that we're doing what we're think we're doing ? could you add a comment at the top of this block explaining whats happening ?